### PR TITLE
Add FAQ page and add section about status page

### PR DESCRIPTION
**Pull request by nicolasarnold12321**
https://deploy-preview-482--roadie.netlify.app/faqs

### CODE REVIEW

File: src/components/landing/faqs.js [hunk 1]
```diff
@@ -8,29 +8,35 @@ const FAQ_CONTENT = [{
...
+  answer:
+    "Yes we do! It can be found here: <a href='https://status.roadie.io'>https://status.roadie.io/</a>",
 }];
 
 
 const Item = ({ question, answer }) => (
   <div>
     <dt className="font-semibold text-gray-900">{question}</dt>
-    <dd className="mt-3 text-gray-500">{answer}</dd>
+    <dd className="mt-3 text-gray-500" dangerouslySetInnerHTML={{ __html: answer }}/>
```
Comment (nicolasarnold12321) [inline src/components/landing/faqs.js:39]: Sounds scary but i think in this case, its not 'dangerous' as we are only displaying simple text

Comment (Xantier) [reply src/components/landing/faqs.js:39]: if you make the answer a function, React should be able to render is as JSX.

Comment (nicolasarnold12321) [reply src/components/landing/faqs.js:39]: Ok cool thanks!

Comment (nicolasarnold12321) [reply src/components/landing/faqs.js]: Done

Comment (Xantier) [reply src/components/landing/faqs.js]: you can do something like 
```
{answer instanceOf Function ? asnwer() : answer}
```
so there's no need to necessarily wrap non-html to be html.

**Review by Xantier — APPROVED** [review]

**Review by Xantier — APPROVED** [review]
aight let's ship it

**Pull request merged**